Detectors/TPC/calibration/SpacePoints/include/SpacePoints/TrackInterpolation.h

```diff
@@ -185,11 +188,23 @@ class TrackInterpolation
 
   // -------------------------------------- processing functions --------------------------------------------------
 
-  /// Initialize everything
-  void init();
+  /// Initialize everything, set the requested track sources
+  void init(o2::dataformats::GlobalTrackID::mask_t src);
+
+  /// Check if input track passes configured cuts
+  bool isInputTrackAccepted(o2::dataformats::GlobalTrackID gid, o2::globaltracking::RecoContainer::GlobalIDSet gidTable, o2::dataformats::PrimaryVertex pv);
```

Comment:
should not be 
```
boolbool isInputTrackAccepted(o2::dataformats::GlobalTrackID gid, o2::globaltracking::RecoContainer::GlobalIDSet gidTable, o2::dataformats::PrimaryVertex pv) isInputTrackAccepted(o2::dataformats::GlobalTrackID gid, o2::globaltracking::RecoContainer::GlobalIDSet& gidTable, o2::dataformats::PrimaryVertex& pv) const
```
?

Reply:
For the not so large data types I thought it might be a premature optimization to pass by const reference instead of by value, but this I can of course change. The method itself is not const, since I am adding the track seeds to the class members in case the track is accepted. But it would be cleaner to do this only after the method has returned true, so I will change that

Reply:
OK for const method, but for any type > 8B long passing by const reference is more economic.